database/PGindex.js

```diff
@@ -0,0 +1,36 @@
+const { Pool} = require('pg');
+// const path = require('path');
+const pool = new Pool({
+  user: 'postgres',
+  host: '127.0.0.1',
+  database: 'SDC',
+  password: '',
+  port: 5432,
+});
+
```

Comment:
It may be beneficial to factor these local variables out for env variables to prep for deployment.

Reply:
Thanks Andrew, will do.